internal/v1/handler_blueprints_test.go

```diff
@@ -346,6 +346,82 @@ func TestHandlers_GetBlueprint(t *testing.T) {
 	require.Equal(t, blueprint.Customizations, result.Customizations)
 }
 
+func TestHandlers_GetImageTypesFromBlueprint(t *testing.T) {
+	ctx := context.Background()
+	dbase, err := dbc.NewDB()
+	require.NoError(t, err)
+
+	db_srv, tokenSrv := startServer(t, &testServerClientsConf{}, &ServerConfig{
+		DBase:            dbase,
+		DistributionsDir: "../../distributions",
+	})
+	defer func() {
+		err := db_srv.Shutdown(ctx)
+		require.NoError(t, err)
+	}()
+	defer tokenSrv.Close()
+
+	id := uuid.New()
+	versionId := uuid.New()
+
+	uploadOptions := UploadRequest_Options{}
+	err = uploadOptions.FromAWSUploadRequestOptions(AWSUploadRequestOptions{
+		ShareWithAccounts: common.ToPtr([]string{"test-account"}),
+	})
+	require.NoError(t, err)
+	name := "blueprint"
+	description := "desc"
+	blueprint := BlueprintBody{
+		Customizations: Customizations{
+			Packages: common.ToPtr([]string{"nginx"}),
+		},
+		Distribution: "centos-8",
+		ImageRequests: []ImageRequest{
+			{
+				Architecture: ImageRequestArchitectureX8664,
+				ImageType:    ImageTypesAws,
+				UploadRequest: UploadRequest{
+					Type:    UploadTypesAws,
+					Options: uploadOptions,
+				},
+			},
+			{
+				Architecture: ImageRequestArchitectureAarch64,
+				ImageType:    ImageTypesAzure,
+				UploadRequest: UploadRequest{
+					Type:    UploadTypesAzure,
+					Options: uploadOptions,
+				},
+			},
+			{
+				Architecture: ImageRequestArchitectureAarch64,
+				ImageType:    ImageTypesGuestImage,
+				UploadRequest: UploadRequest{
+					Type:    UploadTypesAwsS3,
+					Options: uploadOptions,
+				},
+			},
+		},
+	}
+
+	var message []byte
+	message, err = json.Marshal(blueprint)
+	require.NoError(t, err)
+	err = dbase.InsertBlueprint(ctx, id, versionId, "000000", "000000", name, description, message)
+	require.NoError(t, err)
+	respStatusCode, body := tutils.GetResponseBody(t, fmt.Sprintf("http://localhost:8086/api/image-builder/v1/experimental/blueprints/%s/image_types", id.String()), &tutils.AuthString0)
+	require.Equal(t, http.StatusOK, respStatusCode)
+
+	var result []ImageTypes
+	require.Equal(t, 200, respStatusCode)
+	err = json.Unmarshal([]byte(body), &result)
+	require.NoError(t, err)
+	for i, imageType := range result {
+		expectedImageType := blueprint.ImageRequests[i].ImageType
+		require.Equal(t, expectedImageType, imageType)
+	}
```

Comment:
You could use [ElementsMatch](https://pkg.go.dev/github.com/stretchr/testify/require#ElementsMatch), there are many useful functions in that package. But this is okay.